rclcpp/src/rclcpp/node_interfaces/node_graph.cpp

```diff
@@ -134,6 +134,39 @@ NodeGraph::get_service_names_and_types() const
   return services_and_types;
 }
 
+std::map<std::string, std::vector<std::string>>
+NodeGraph::get_service_names_and_types_by_node(
+  const std::string & node_name,
+  const std::string & namespace_) const
+{
+  rcl_names_and_types_t service_names_and_types {};
```

Comment:
I think we should call `rcl_get_zero_initialized_names_and_types()`, which does an [explicit initialization of the members](https://github.com/ros2/rmw/blob/d030ad19eaeb6bee3eda7c0197360442e2db0a7a/rmw/src/names_and_types.c#L26-L29). I think this avoids warnings on some compilers.

Reply:
Good point - done: 435a24f9aa1c35a545eb6d7f2cd9876dc93759ec.